index.html

```diff
@@ -1910,39 +1919,44 @@ <h3>Add Proof</h3>
       <section>
         <h3>Add Proof Set/Chain</h3>
         <p>
-          The following algorithm specifies how to incrementally add a proof to
+          The following algorithm specifies how to incrementally add a proof to 
           a proof set or proof chain starting with a secured document containing
           either a proof or proof set/chain.
-          Required inputs are a <a>secured data document</a>
-          (<var>securedDocument</var>) and <a>proof options</a>
+          Required inputs are a <a>secured data document</a> 
+          (<var>securedDocument</var>) and <a>proof options</a> 
           (<var>options</var>). The <a>proof options</a>
-          (<var>options</var>) must satisfy the criteria in
+          (<var>options</var>) must satisfy the criteria in 
           section <a href="#add-proof"></a>.
-          A new <a>secured data document</a> is produced as
-          output. Whenever this algorithm encodes strings, it MUST use UTF-8
+          A new <dfn>secured data document</dfn> is produced as
+          output. Whenever this algorithm encodes strings, it MUST use UTF-8 
           encoding.
         </p>
 
         <ol class="algorithm">
           <li>
-Let <var>proof</var> be set to <var>securedDocument</var>.<var>proof</var>. Let
-<var>proof_list</var> be an empty list. If <var>proof</var> is a list copy all
-the elements of <var>proof</var> to <var>proof_list</var>. If <var>proof</var>
+Let <var>proof</var> be set to <var>securedDocument</var>.<var>proof</var>. Let 
+<var>proof_list</var> be an empty list. If <var>proof</var> is a list copy all 
```

Comment:
```suggestion
<var>allProofs</var> be an empty list. If <var>proof</var> is a list copy all 
```

All algorithms use camelCase. Don't care much about the name, but at least camelCase it, please.